src/modules/logic-analyzer/CoherenceValidator.ts

```
@@ -0,0 +1,742 @@
+      earlier: 1,
+      previously: 1,
+      prior: 1,
+      during: 2,
+      while: 2,
+      meanwhile: 2,
+      simultaneously: 2,
+      after: 3,
+      later: 3,
+      subsequently: 3,
+      following: 3,
+      then: 4,
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)